modules/platforms/cpp/ignite/tuple/tuple_test.cpp

```diff
@@ -26,6 +26,7 @@
 #include <tuple>
 #include <type_traits>
 #include <vector>
+#include <optional>
```

Comment:
The rest of the file has no changes, is this needed?

Reply:
Yeah. It seems that I removed `<optional>` from some other header that was included in this one and everything stopped compiling. It's better to explicitly include all used standard headers in the file anyway.